<ui version="4.0" >
 <class>WdgDisplaySettings</class>
 <widget class="QWidget" name="WdgDisplaySettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Display</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="grpOpenGL" >
     <property name="title" >
      <string>OpenGL</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="cbUseOpenGL" >
        <property name="text" >
         <string>Enable OpenGL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbUseOpenGLShaders" >
        <property name="text" >
         <string>Enable OpenGL shaders</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="grpCheckers" >
     <property name="title" >
      <string>Transparency Checks</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="KColorButton" name="colorChecks" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblColor" >
        <property name="text" >
         <string>Checkers Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="chkMoving" >
        <property name="toolTip" >
         <string>If checked, the checkers will move when scrolling the canvas.</string>
        </property>
        <property name="whatsThis" >
         <string>Determines whether the checks will stay put or whether they will scroll together with the canvas</string>
        </property>
        <property name="text" >
         <string>&amp;Move checkers when scrolling</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="intCheckSize" >
        <property name="maximum" >
         <number>256</number>
        </property>
        <property name="value" >
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblCheckSize" >
        <property name="text" >
         <string>&amp;Checkers size</string>
        </property>
        <property name="buddy" >
         <cstring>intCheckSize</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>356</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
